include/acceleration.hpp

```diff
@@ -8,16 +8,14 @@
 
 namespace obc {
 
-using Accelerometer = MMA8452Q;
```

Comment:
czemu rezygnujemy z tego "utajnienia" czym jest dokładnie accelerometr?

Reply:
Ludzie porobili aliasy na wszystkie typy z bilbiotek zewnętrznych: MMA8452Q, Adafruit_GPS, BMP280 i nmea_float_t. Nie widzę w tym żadnych plusów, jedynie mamy więcej kodu i pośredników. W idealnym świecie chcielibyśmy, zeby te biblioteki miały własne przestrzenie nazw i żeby źródło ich pochodzenia było jak najbardziej jawne.

Jeśli chodzi o jakieś potencjalne możliwości abstrakcji konkretnego modelu sprzętu... no samym aliasem tego nie zrobisz :P

Początkowo użyłem aliasu ```using Accelerometer = MMA8452Q```, dlatego że oryginalna nazwa komponentu nie wskazywała na to czym on jest. W tym momencie można to odczytać z kontekstu, np z nazw zmiennych, dlatego nie jest on raczej potrzebny.